{"uuid": "9635494e-fe0b-4ebe-bc01-67307a7d5405", "children": ["f2ef67a9-712c-44d5-ab8f-d6774b7af638"], "befores": [{"name": "yzm", "status": "broken", "statusDetails": {"message": "NameError: name 'TestFaker' is not defined\n", "trace": "  File \"D:\\ruanjian\\python\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\ruanjian\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\ruanjian\\python\\lib\\site-packages\\_pytest\\fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\daima\\xin-daima\\two-pytest-api-automation\\case\\conftest.py\", line 69, in yzm\n    phone = TestFaker().get_phone_number()\n"}, "start": 1638435333750, "stop": 1638435333750}], "start": 1638435333750, "stop": 1638435333816}